src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionBatchFixAllProvider.cs

```diff
@@ -50,8 +47,8 @@ internal abstract class AbstractSuppressionBatchFixAllProvider : FixAllProvider
             var cancellationToken = fixAllContext.CancellationToken;
             if (documentsAndDiagnosticsToFixMap?.Any() == true)
             {
-                var progressTracker = fixAllContext.GetProgressTracker();
-                progressTracker.Description = fixAllContext.GetDefaultFixAllTitle();
+                var progressTracker = fixAllContext.ProgressTracker;
+                progressTracker.Report(CodeAnalysisProgress.Description(fixAllContext.GetDefaultFixAllTitle()));
```

Comment:
here's an example of using the new API versus the old one.  Here we're 'reporting' a new description to show to the user.  This is the IProgress way of doing things.  Everything is a 'report' and you package up the data change into an appropriate 'progress' nugget that ends up flowing to the UI to eventually change.